src/RestApiHelpers/Validation/ValidateActionParametersAttribute.cs

```diff
@@ -52,10 +65,16 @@ private void EvaluateValidationAttributes(ParameterInfo parameter, object argume
 
                 if (validationAttribute != null)
                 {
-                    var isValid = validationAttribute.IsValid(argument);
-                    if (!isValid)
+                    var validationResult = validationAttribute
+                        .GetValidationResult(
+                            validationContext.ObjectInstance, 
+                            validationContext);
+                    
+                    if (!ReferenceEquals(validationResult, ValidationResult.Success))
```

Comment:
Please use just simple `!==`, that's easier to read, plus the framework seems to be using it too (for example here: https://github.com/aspnet/Mvc/blob/24eaa740f5b1736700d8d91053f60d690f4fc17e/src/Microsoft.AspNetCore.Mvc.DataAnnotations/Internal/ValidatableObjectAdapter.cs#L51).